Darwin/Services/StitchRemoteMongoDBService/StitchSyncPerformanceTests/SyncPerformanceIntTestHarness.swift

```diff
@@ -79,167 +76,133 @@ class SyncPerformanceIntTestHarness: BaseStitchIntTestCocoaTouch {
         outputMongoClient = try! outputClient?.serviceClient(fromFactory: remoteMongoClientFactory,
                                                              withName: "mongodb-atlas")
         outputColl = outputMongoClient?.db(stitchOutputDbName).collection(stitchOutputCollName)
+
+        return true
     }
 
     override func setUp() {
         super.setUp()
     }
 
-    // swiftlint:disable function_body_length
-    func runPerformanceTestWithParameters(testParams: TestParams,
-                                          testDefinition: TestDefinition,
-                                          beforeEach: SetupDefinition = { _, _, _ in },
-                                          afterEach: TeardownDefinition = { _, _, _ in }) {
-        var failed = false
-        setupOutputClient()
+    var performanceTestingContext: SyncPerformanceTestContext? {
```

Comment:
[q/style] this creates a new performance testing context every time you access this property. Is that the behavior you want? if it is, then I think this makes more sense as a method than a computed property. Computed properties are more for computing some sort of state rather than producing new objects.

Reply:
This is the desired behavior. I literally just changed it to a computer property, but I'll change it back to a function